### PR TITLE
feat: allow multiple position in a workplace

### PR DESCRIPTION
**Pull request by mangkoran**
This PR adds support for multiple position/team in one workplace. Could you please recheck the updated JSON schema? I am afraid I missed one or two 😅

### CODE REVIEW

File: cv.typ.schema.json [hunk 1]
```diff
@@ -1,1854 +1,540 @@
...
-                }
-            ]
+            "items": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    }
+  },
```
Comment (jskherman) [inline cv.typ.schema.json:335]: Could you remove this extra bracket here please and move it to the end of the file? It closes the properties field too early and makes the subsequent fields like certificates be in the top-level.

File: example.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# yaml-language-server: $schema=https://raw.githubusercontent.com/jskherman/cv.typ/main/cv.typ.schema.json
+# yaml-language-server: $schema=./cv.typ.schema.json
```
Comment (jskherman) [inline example.yml:1]: Could we also return the original URL here please? I don't think this will work with submodules.

Comment (jskherman) [inline example.yml:1]: Could we also return the original URL here please? The value here is relative to the root and, thus, won't work with submodules.

Comment (jskherman) [inline example.yml:1]: Could we also return the original URL here please? The value here is relative to the root and, thus, won't work with submodules.

File: example.yml [hunk 3]
```diff
@@ -109,7 +118,7 @@ awards:
     url:
     location: City of London, United Kingdom
     highlights:
-      # - Champion in the Division A District X Speech Evaluation Contest of Toastmasters International in London.
+      - Champion in the Division A District X Speech Evaluation Contest of Toastmasters International in London.
```
Comment (jskherman) [inline example.yml:121]: Could you also please delete this line? I forgot I had this still commented out to test awards without a description.

**Review by jskherman — CHANGES_REQUESTED** [review]
Thank you so much for your significant contribution again @mangkoran. I think we can merge this already once some small changes are done.

File: example.yml (hunk 2 above)
Comment (mangkoran) [reply example.yml:1]: Sorry I forgot to revert it when I debugging the schema.

File: example.yml (hunk 3 above)
Comment (mangkoran) [reply example.yml:121]: Alright I will do that.

**Comment by mangkoran** [comment]
@jskherman I have updated the PR. Could you please check it again? Thank you.

Anyway, actually I'd like to reorder the section so Work -> Education instead of Education -> Work (which I felt a bit weird since we have the `example.yml` and Typst output arranged differently 😅). What's you preference: should I just add the commit in this PR or create a separate PR?

File: example.yml [hunk 4]
```diff
@@ -108,8 +117,6 @@ awards:
     issuer: Toastmasters International
     url:
     location: City of London, United Kingdom
-    highlights:
```
Comment (jskherman) [inline example.yml:111]: This `highlights` fields still needs to be present with a null value in order for the template to compile.

```
highlights: 
```

**Comment by jskherman** [comment]
> @jskherman I have updated the PR. Could you please check it again? Thank you.
> 
> Anyway, actually I'd like to reorder the section so Work -> Education instead of Education -> Work (which I felt a bit weird since we have the `example.yml` and Typst output arranged differently 😅). What's you preference: should I just add the commit in this PR or create a separate PR?

It could really go either way for me, but Work => Education makes more sense.

File: example.yml (hunk 4 above)
Comment (mangkoran) [reply example.yml:111]: Sorry my bad. Will update it.

**Comment by mangkoran** [comment]
Updated with work and education switched.

**Comment by mangkoran** [comment]
I think that's all for now. Thank you for your help!

**Pull request merged**